### PR TITLE
adr reporting review

### LINKED ISSUE #168: Review 05-Reporting

### PR DESCRIPTION
**Pull request by kusweta**
Fix #168 
ADR Review: 
Reporting

### CODE REVIEW

File: data-reconciliation-app/docs/adr/05-Reporting.md [hunk 1]
```diff
@@ -6,30 +6,30 @@ Proposed
 
 ## Context
 
-We need to build an API that reports the members/users' reconciled dataset back to them.
+We need to build an API that reports the members' and users' reconciled datasets back to them.
```
Comment (mkcomer) [inline data-reconciliation-app/docs/adr/05-Reporting.md:9]: nit: change datasets to records....as they can report on individual or multiple records

File: data-reconciliation-app/docs/adr/05-Reporting.md [hunk 2]
```diff
@@ -92,23 +89,23 @@ getDataByKey(userId: string, key: string) : SummaryResult   // for a single reco
 
 ### Summary Result Object
 
-The SummaryResult Object will be the final output object in the report.
+The SummaryResult object will be the final output in the report. In the report, the key and value will match the schema defined.
```
Comment (mkcomer) [inline data-reconciliation-app/docs/adr/05-Reporting.md:92]: maybe add a link to the code where this schema is defined?

**Review by mkcomer — APPROVED** [review]
well done, nice clean up @kusweta. Only some small nits/comments

File: data-reconciliation-app/docs/adr/05-Reporting.md (hunk 2 above)
Comment (kusweta) [reply data-reconciliation-app/docs/adr/05-Reporting.md]: link added

**Review by julioalex-rezende — APPROVED** [review]

**Pull request merged**